### ISSUE TITLE
Provide way to easily delete backups

### ISSUE THREAD

**Issue by SamuelMiller** [getflywheel/local-addon-backups#56]
Currently, there is no easy way to delete google drive backups. Delete the backup files on Google drive does remove the backup list and descriptions in the Local WP appl

**Comment by realanthonyc** [comment]
Agree. Need a way to identify which backup "folder" to be deleted on Dropbox. There is no way to delete backups, even an instance has been removed completely.

**Comment by sramekmichal** [comment]
Agree and even after deletion of all files in the folder Apps, the plugin somehow remember old backups and cannot upload a new one.

This is really painful 🤕
